website/docs/modules/ROOT/nav.adoc

```diff
@@ -95,7 +95,6 @@
 *** xref:contrib/proguard.adoc[]
 *** xref:contrib/scalapblib.adoc[]
 *** xref:contrib/scoverage.adoc[]
-*** xref:contrib/sonatypecentral.adoc[]
```

Comment:
We should either keep the old page as a forwarder to the new, with a notice that this plugin moved into `scalalib`. Or we should at least add a `page-alias` to the new page, so the old link still works.